RELEASING.md

```diff
@@ -18,17 +18,32 @@ To bump all (execpt the `/build` module's) dependencies run:
    if necessary.
    Contact @splunk/gdi-docs team if needed.
 
-2. Run the pre-release script which updates go.mod for submodules to depend on
-   the new release. It creates a branch `pre_release_<new tag>`
-   that will contain all release changes.
+1. Update the version in [`versions.yaml`](versions.yaml)
+
+1. Run the pre-release step which updates `go.mod` and `version.go` files
+   in modules for the new release.
+
+    ```sh
+    ./goyek.sh prerelease
+    ```
+
+1. Checkout to the branch create by `multimod`.
+
+1. Sign the commit created by `multimod`.
+
+    ```sh
+    git commit --amend --no-edit -s
+    ```
+
```

Comment:
Instead of signing a commit by amending upstream just merges the commit into the release branch.

```suggestion
1. Create a new release branch. I.e. `git checkout -b release-X.X.X main`.
1. Update the version in [`versions.yaml`](versions.yaml)

1. Run the pre-release step which updates `go.mod` and `version.go` files
   in modules for the new release.

    ```sh
    ./goyek.sh prerelease
    ```

1. Merge the branch create by `multimod` into your release branch.
```